### PR TITLE
Added Meteor blog post.

### PR DESCRIPTION
**Pull request by frankdiox**
@argelius Do you think this is enough?

### CODE REVIEW

**Comment by frankdiox** [comment]
@argelius @masahirotanaka What do you think about the last update?
I'm not sure about the title and category. Right now it's `tutorial` but it sort of feels like `announcement` after the last change.

File: blog/posts/2016-08-23.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+author: fran
+date: 2016-08-23
+id: "meteor-mobile-app-onsenui"
+title: "Meteor Mobile UI framework powered by Onsen UI"
```
Comment (masahirotanaka) [inline blog/posts/2016-08-23.markdown]: Suggestion: Getting Started with Meteor Framework and Onsen UI
(Background idea: I'd like to inform the reader that this article is a tutorial content)

File: blog/posts/2016-08-23.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+This last characteristic is what makes it pluggable to Onsen UI. Being framework-agnostic, Onsen UI is future proof and can easily play well with famous technologies such as Meteor. This way, we keep a quick learning curve as well as high efficiency since Onsen UI core is implemented in pure JavaScript. Moreover, it offers beautiful looking components that change style automatically depending on the platform (iOS flat design or Android's Material Design), making cross-development truly possible with little to no effort.
+
+
+## Onsen UI + Meteor ToDo app
+
+As already mentioned, Meteor offers variety of frameworks for its front-end side. Two of them, React and AngularJS, are already tested and play nicely with Onsen UI since we released official components for them. Here we will show how easy is to integrate [React Components for Onsen UI](https://onsen.io/v2/react.html) with your Meteor application.
```
Comment (masahirotanaka) [inline blog/posts/2016-08-23.markdown]: Suggestion: make "already tested and play nicely with Onsen UI" in border text.
(Background idea: The most important statement in this blog post).

File: blog/posts/2016-08-23.markdown [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+We follow the Meteor's official tutorial to make a [React ToDo app](https://www.meteor.com/tutorials/react/creating-an-app) and simply change the front end to use Onsen UI components. This app runs on [Heroku](https://onsenui-meteor-todos.herokuapp.com/).
```
Comment (masahirotanaka) [inline blog/posts/2016-08-23.markdown]: Suggestion: "React ToDo app" to "ToDo app written in React"
(Background idea: I feel a bit strange when React is used as adjective in this context)

**Comment by argelius** [comment]
Thanks @frankdiox !

**Pull request merged**